lang/cs/Org.Apache.REEF.Client/Org.Apache.REEF.Client.csproj

```diff
@@ -27,6 +27,7 @@ under the License.
     <FileAlignment>512</FileAlignment>
     <RestorePackages>true</RestorePackages>
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..</SolutionDir>
+    <TempResxFile Condition="$(TempResxFile) == '' Or $(TempResxFile) == '*Undefined*'">$(SolutionDir)Org.Apache.REEF.Client\Properties\Resources.resx</TempResxFile>
```

Comment:
Shouldn't there be a `\` here, as in: `$(SolutionDir)\Org.Apache.REEF.Client\Properties\Resources.resx`